packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -157,7 +188,29 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
                     <div className={`adyen-checkout__dropin adyen-checkout__dropin--${status.type}`}>
                         {isRedirecting && status.props.component && status.props.component.render()}
                         {isLoading && status.props && status.props.component && status.props.component.render()}
-                        {!!hasPaymentMethodsToBeDisplayed && (
+
+                        {showFastlane && (
+                            <Fragment>
+                                <PaymentMethodList
+                                    isLoading={isLoading}
+                                    paymentMethods={fastlanePaymentElement}
+                                    activePaymentMethod={activePaymentMethod}
+                                    cachedPaymentMethods={cachedPaymentMethods}
+                                    onSelect={this.handleOnSelectPaymentMethod}
+                                    openFirstPaymentMethod
+                                    showRadioButton={this.props.showRadioButton}
+                                />
+                                <Button
+                                    classNameModifiers={['dropin-show-paymentmethods']}
+                                    variant="link"
+                                    inline
+                                    label="Other payment methods"
+                                    onClick={this.onShowDefaultPaymentMethodListClick}
+                                />
+                            </Fragment>
+                        )}
+
+                        {!!hasPaymentMethodsToBeDisplayed && !showFastlane && (
```

Comment:
Can be then modified to `!!hasPaymentMethodsToBeDisplayed && showDefaultPaymentMethodList`?